lib/git-lfs-s3/application.rb

```diff
@@ -47,6 +49,151 @@ def protected!
       "Git LFS S3 is online."
     end
 
+    def valid_object?(object)
+      begin
+        valid = object[:size] >= 0
+      rescue
+        valid = false
+      end
+      begin        
+        if valid
+          oid = object[:oid].hex
+          valid = oid.size == 32 && object[:oid].size == 64
+        end
+      rescue
+        valid = false
+      end
+      valid
+    end
```

Comment:
I'm verifying that the object is a sha256 hash. I figured there would be a digest method that would do this but I didn't find one. This logic is off too, oid.hex.size <= 32. Because a hash like this is possible "00f77c4030a291cca5884c6a6e72f5f648704f229f74092ba8b8f33f3b8dab09z" which has a size of 31 and is a valid hash.